### PR TITLE
Create Dashboard page

### PR DESCRIPTION
**Pull request by phlgr**
Create first draft of dashboard page based on components from last week.
Please take a close look at the great guide to grid: https://css-tricks.com/snippets/css/complete-guide-grid/

### CODE REVIEW

**Review by ChristianSeelemann — APPROVED** [review]

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  --clr-secondary: #161060;
+  --clr-text-primary: #393939;
+  --clr-text-secondary: #111;
+  --clr-text-passive: hsla(0, 0%, 0%, 0.55);
+  --clr-content-passive: #fbfbfb;
+  --fw-primary: 400;
+  --fw-secondary: 600;
+}
+
+body {
```
Comment (marwinburesch) [inline styles.css:22]: I think we should set `color` here to `--clr-text-primary`, I forgot to do that in the `global.css` for the components. There is an issue for that #17

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  margin: 0;
+  font-family: var(--ff-primary);
+}
+
+.container {
+  display: grid;
+  height: 100%;
+  grid-template-columns: 1fr 5fr;
+  grid-template-rows: 1fr 1fr 4fr 1fr;
+}
```
Comment (marwinburesch) [inline styles.css:43]: a linebreak here maybe? :)

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Looks awesome

**Review by marwinburesch — APPROVED** [review]
cheers bud

**Pull request merged**